unearthed-arcana/2022/20220308.xml

```diff
@@ -0,0 +1,972 @@
+		</sheet>
+	</element>
+
+	<element name="Crescent Moon" type="Archetype Feature" source="Unearthed Arcana: Heroes of Krynn" id="ID_WOTC_UA20220308_ARCHETYPE_FEATURE_LUNAR_MAGIC_LUNAR_EMBODIMENT_CRESCENT_MOON">
+		<supports>UA Lunar Embodiment</supports>
+		<description>
+			<p>You manifest the crescent moon.</p>
+		</description>
+		<sheet>
+			<description>You can cast sanctuary once without expending a spell slot.</description>
+			<description level="3">You can cast sanctuary and blindness/deafness once each without expending a spell slot.</description>
+			<description level="5">You can cast sanctuary, blindness/deafness, and phantom steed once each without expending a spell slot.</description>
+			<description level="7">You can cast sanctuary, blindness/deafness, phantom steed, and hallucinatory terrain once each without expending a spell slot.</description>
+			<description level="9">You can cast sanctuary, blindness/deafness, phantom steed, hallucinatory terrain, and dream once each without expending a spell slot.</description>
+		</sheet>
+	</element>
+
+	<element name="Lunar Boons" type="Archetype Feature" source="Unearthed Arcana: Heroes of Krynn" id="ID_WOTC_UA20220308_ARCHETYPE_FEATURE_LUNAR_MAGIC_LUNAR_BOONS">
+		<description>
+			<p><em>6th-Level Lunar Magic Feature</em></p>
+			<p>Lunar phases influence your metamagic. Each lunar phase is associated with spells of the following schools of magic:</p>
+			<p class="indent"><strong><em>Full Moon.</em></strong> Abjuration and conjuration spells</p>
+			<p class="indent"><strong><em>New Moon.</em></strong> Evocation and necromancy spells</p>
+			<p class="indent"><strong><em>Crescent Moon.</em></strong> Divination and transmutation spells</p>
+			<p class="indent">Whenever you use Metamagic on a spell from the Lunar Spells table or a spell of a school of magic associated with the lunar phase you are in, you can reduce the sorcery points spent by 1 (minimum 0). You can reduce the sorcery points spent for your Metamagic a number of times equal to your proficiency bonus, and you regain all expended uses when you finish a long rest.</p>
+		</description>
+		<sheet usage="{{proficiency}}/Long Rest">
+			<description>Whenever you use Metamagic on a spell from the Lunar Spells table or a spell from the {{ua:lunar boon:schools}} school of magic, you can reduce the sorcery points spent by 1 (minimum 0).</description>
+		</sheet>
+		<rules>
+			<stat name="ua:lunar boon:schools" value="abjuration or conjuration" inline="true" requirements="ID_WOTC_UA20220308_ARCHETYPE_FEATURE_LUNAR_MAGIC_LUNAR_EMBODIMENT_FULL_MOON"/>
+			<stat name="ua:lunar boon:schools" value="evocation or necromancy" inline="true" requirements="ID_WOTC_UA20220308_ARCHETYPE_FEATURE_LUNAR_MAGIC_LUNAR_BOONS_NEW_MOON"/>
+			<stat name="ua:lunar boon:schools" value="divination or transmutation" inline="true" requirements="ID_WOTC_UA20220308_ARCHETYPE_FEATURE_LUNAR_MAGIC_LUNAR_BOONS_CRESCENT_MOON"/>
```

Comment:
also just checked and inline stats do get stuck and can't be changed when you swap between phases. It's the same issue i had with Dragonborn Revenant, where either your original damage type gets stuck or necrotic gets stuck, app just doesn't know what to do when it needs to change inlines, so i think we should change how it works.